### PR TITLE
pflask

### PR DESCRIPTION
**Pull request by tg-x**

### CODE REVIEW

File: pkgs/os-specific/linux/pflask/default.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+  src = fetchurl {
+    url = "https://github.com/ghedo/pflask/archive/${rev}.tar.gz";
+    sha256 = "2545fca37f9da484b46b6fb5e3a9bbba6526a9725189fe4af5227ef6e6fca440";
+  };
+
+  buildInputs = [ python ];
+
+  configurePhase = ''
+    ln -s ${fetchurl {
```
Comment (domenkozar) [inline pkgs/os-specific/linux/pflask/default.nix]: we have `waf-1.8.12` in nixpkgs, could we use that one?

Comment (tg-x) [inline pkgs/os-specific/linux/pflask/default.nix]: possibly, but I don't see any waf package..
(the `mpv` package also fetches it similarly, others bundle waf with the code)

Comment (domenkozar) [inline pkgs/os-specific/linux/pflask/default.nix]: Yeah we need to stop doing that. Package waf properly and reuse it.

**Comment by tg-x** [comment]
added a `waf` package

**Pull request merged**